tilequeue/metatile.py

```diff
@@ -16,6 +18,10 @@ def make_single_metatile(size, tiles):
     if len(tiles) == 0:
         return []
 
+    if date_time is None:
+        from time import gmtime
```

Comment:
Good point! I had picked up the wrong habit from the tests. And today I learned that there's a Python style thing that Flake8 _doesn't_ complain about :wink: